frontend/src/views/WheresMyGeneV2/components/HeatMap/index.tsx

```diff
@@ -122,7 +126,32 @@ export default memo(function HeatMap(props: Props): JSX.Element {
               }
             )}
           </YAxisWrapper>
-          <ChartWrapper ref={chartWrapperRef} top={xAxisHeight}>
+          {isLoadingAPI || isAnyTissueLoading(isLoading) ? (
+            <ChartWrapper
+              top={xAxisHeight}
+              hidden={!isAnyTissueLoading(isLoading)}
+            >
+              <SkeletonContainer>
+                {[...Array(totalElementsCount)].map((i) => (
```

Comment:
```suggestion
                {[...Array(totalElementsCount)].map((_, index) => (
```
if the intention is to use the array `index`! Otherwise I think currently `i` is `undefined`?